.github/workflows/nightly.yml

```diff
@@ -23,11 +23,11 @@ on:
 
 jobs:
   build-test-package:
-    name: 'Build, Test, and Package (OS: ${{ matrix.os }}, Node: ${{ matrix.node }}, Java: ${{ matrix.java_version }}, VS Code: ${{ matrix.vscode }} )'
+    name: 'Build, Test, and Package (OS: ${{ matrix.os }}, Node: ${{ matrix.node }}, Java: ${{ matrix.java_version }}, VS Code: ${{ matrix.vscode }})'
     strategy:
       matrix:
         os: [ macos-12, ubuntu-20.04, windows-2019, macos-latest, ubuntu-latest, windows-latest ]
-        node: [ '16', '18.20.1' ] # version 18.20.2 (current latest of 18) is broken on windows
```

Comment:
Add 16 back, nodejs 16 is our minimal supported version so we want to make sure to test it in CI.